Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -42,8 +42,9 @@ def cmd_output(*args)
     cmd_args << "-rintegration_mocks"
     cmd_args << (HOMEBREW_LIBRARY_PATH/"../brew.rb").resolved_path.to_s
     cmd_args += args
+    cmd_exec = [RUBY_PATH] + cmd_args
     Bundler.with_original_env do
-      ENV["HOMEBREW_BREW_FILE"] = HOMEBREW_PREFIX/"bin/brew"
+      ENV["HOMEBREW_BREW_FILE"] = cmd_exec.join " "
```

Comment:
I don't think this is going to work and, more importantly, I don't think we want to do this. `HOMEBREW_BREW_FILE` should be a path to the `bin/brew` script, not a loose collection of arguments to be interpreted by a shell. I think fixing nested `brew` invocations in integration tests definitely needs some more thought and likely a separate PR.